### PR TITLE
Enable embedded boundary tests

### PR DESCRIPTION
**Pull request by RemiLehe**
It seems that the CI test for the embedded boundaries was not activated do to a missing end-of-line character. 
This PR fixes this. I also changed the permissions on the corresponding Python analysis script.

### CODE REVIEW

File: Regression/WarpX-tests.ini [hunk 1]
```diff
@@ -2017,4 +2017,4 @@ numthreads = 1
 compileTest = 0
 doVis = 0
 compareParticles = 0s
```
Comment (RevathiJambunathan) [inline Regression/WarpX-tests.ini]:
```suggestion
compareParticles = 0
```

Comment (RevathiJambunathan) [reply Regression/WarpX-tests.ini]: That would either be 0/1?

**Comment by EZoni** [comment]
I also noticed that we have
https://github.com/ECP-WarpX/WarpX/blob/e6555d79da7374b024b25f2ebdef2e14fe7c1c7e/Regression/WarpX-tests.ini#L2014
and since it's safer to run on up to 2 parallel processes on the CI platforms, I would suggest that we set `numprocs = 2` instead. 

P.S.
Note that `numthreads` is overwritten in `prepare_file_travis.py`, but `numprocs` is not (because we do want to be able to run some tests on 1 MPI process and some others on 2 MPI processes):
https://github.com/ECP-WarpX/WarpX/blob/e6555d79da7374b024b25f2ebdef2e14fe7c1c7e/Regression/prepare_file_travis.py#L75-L76

**Comment by RemiLehe** [comment]
@RevathiJambunathan @EZoni Thanks a lot for your comments! I updated the test, but now it seems that it fails for some reason.
@lgiacome Would you have time to look into why this test is failing?

**Comment by lgiacome** [comment]
I will definitely look into it as soon as possible next week. I think it was running ok for me (locally on my machine) but I will have a closer look.

**Comment by RemiLehe** [comment]
Looking at this closer, it seems that, previously, the `embedded_cube_boundary` test *did* run, but that it did not call the corresponding Python analysis script:
Screen Shot 2021-05-03 at 9 04 46 AM
With this PR, the Python analysis script is now called, but unfortunately fails.

**Comment by RemiLehe** [comment]
From an offline discussion with @lgiacome : it seems that this bug is now fixed in #1919

**Pull request closed without merging**